execute/flist.designware

```diff
@@ -0,0 +1,15 @@
++incdir+$DESIGNWARE_PATH/sim_ver/
+$DESIGNWARE_PATH/sim_ver/DW03_pipe_reg.v
+$DESIGNWARE_PATH/sim_ver/DW_mult_pipe.v
+$DESIGNWARE_PATH/sim_ver/DW_div_pipe.v
+$DESIGNWARE_PATH/sim_ver/DW_fp_add.v
+$DESIGNWARE_PATH/sim_ver/DW_fp_sub.v
+$DESIGNWARE_PATH/sim_ver/DW02_mult.v
+$DESIGNWARE_PATH/sim_ver/DW_div.v
+$DESIGNWARE_PATH/sim_ver/DW_fp_addsub.v
+$DESIGNWARE_PATH/sim_ver/DW_fp_mult.v
+$DESIGNWARE_PATH/sim_ver/DW_fp_div.v
+$DESIGNWARE_PATH/sim_ver/DW_fp_sqrt.v
+$DESIGNWARE_PATH/sim_ver/DW_fp_flt2i.v
+$DESIGNWARE_PATH/sim_ver/DW_fp_cmp.v
+/usr/licensed/synopsys/syn_vN-2017.09-SP4/dw/sim_ver/DW_fp_i2flt.v
```

Comment:
I think I fixed this back to DESIGNWARE_PATH in the main flist? We should fix it here to be consistent

Reply:
These lines are the exact copy of those from the [main flist](https://github.com/bring-your-own-core/anycore-riscv-src/blob/piton/flist). I just moved them to a different file, and under the execute folder. This was done to make it easier to leave out. Also added this to sims [here](https://github.com/bring-your-own-core/byoc/blob/b802d53d5bc85b3b91dd126523e72c2c2f06b1a3/piton/tools/src/sims/manycore.config#L63-L65).

Reply:
I'm saying to fix the last path so it says `$DESIGNWARE_PATH/sim_ver/DW_fp_i2flt.v` instead